Sources/StreamChatUI/ChatMessageList/Attachments/ChatMessageLinkPreviewView.swift

```diff
@@ -116,7 +116,11 @@ open class ChatMessageLinkPreviewView: _Control, ThemeProvider {
 
         authorLabel.textColor = tintColor
 
-        imagePreview.loadImage(from: payload?.previewURL, components: components)
+        components.imageLoader.loadImage(
+            into: imagePreview,
+            url: payload?.previewURL,
+            imageCDN: components.imageCDN
+        )
```

Comment:
Hm, we loose resizing here, don't we?
```suggestion
        components.imageLoader.loadImage(
            into: imagePreview,
            url: payload?.previewURL,
            imageCDN: components.imageCDN,
            resize: true
        )
```

Reply:
Changed the default behavior to the implementation we had previously 👍